obs-studio-server/source/osn-audio.cpp

```diff
@@ -146,4 +165,122 @@ void osn::Audio::SetLegacySettings(void *data, const int64_t id, const std::vect
 
 	rval.push_back(ipc::value((uint64_t)ErrorCode::Ok));
 	AUTO_DEBUG;
+}
+
+void osn::Audio::GetMonitoringDevice(void *data, const int64_t id, const std::vector<ipc::value> &args, std::vector<ipc::value> &rval)
+{
+	rval.push_back(ipc::value((uint64_t)ErrorCode::Ok));
+	const char *name = "";
+	const char *idDevice = "";
+	obs_get_audio_monitoring_device(&name, &idDevice);
+	if (name && idDevice) {
```

Comment:
This check will not work as intended as obs_get_audio_monitoring_device not gonna set pointers to NULL. it just leave them unchanged.